pkg/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', 'build/x86_64_xenial' ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'run', '-v', '/workspace:/workspace', '-w', '/workspace/src/',  'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', '/bin/bash', '-c', 'make purge && make -j4 DISTRO=xenial deb' ]
```

Comment:
Instead of using bracketed arrays, do you want to change it to one argument per line, e.g.:
```yaml
- name: 'gcr.io/cloud-builders/docker'
  args:
  - run
  - -v
  - /workspace:/workspace
  - -w
  - /workspace/src
  - 'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container'
  - /bin/bash
  - -c
  - 'make purge && make -j4 DISTRO=xenial deb'
```
?